yt_dlp/postprocessor/__init__.py

```diff
@@ -33,15 +33,38 @@
 from .sponskrub import SponSkrubPP
 from .sponsorblock import SponsorBlockPP
 from .xattrpp import XAttrMetadataPP
-from ..plugins import load_plugins
+from .._globals import plugin_pps, postprocessors
+from ..plugins import PACKAGE_NAME, register_plugin_spec, PluginSpec
+from ..utils import deprecation_warning
 
-_PLUGIN_CLASSES = load_plugins('postprocessor', 'PP')
+
+def __getattr__(name):
+    lookup = plugin_pps.value
+    if name in lookup:
+        deprecation_warning(
+            f'Importing a plugin Post-Processor from {__name__} is deprecated. '
+            f'Please import {PACKAGE_NAME}.postprocessor.{name} instead.')
+        return lookup[name]
+
+    raise AttributeError(f'module {__name__!r} has no attribute {name!r}')
```

Comment:
do we not want a `.postprocessors` submodule similar to the `extractor.extractors`? Ref: https://github.com/yt-dlp/yt-dlp/pull/7426/files#diff-48582c7e10f56ce857f222b82f43a932119f417bc4ccc8730b0c46a18a6ddc46

Reply:
I think exposing a `postprocessors` submodule could now be useful, given we can use `__get__` to lazily load the postprocessors inclusing wrapped plugins without any problems. The main reason I am always against it is that it encourages importing from a parent module, instead of the proper submodule.

Though if done right, this can ease developer experience... I'm not sure which I prefer.

Reply:
> The main reason I am always against it is that it encourages importing from a parent module, instead of the proper submodule.

I can see your point for normal imports. But for lazy imports, is that really a problem?

I believe having one place to import everything from would be best. API users typically should not have to worry about the submodule structure.

Reply:
Yeah I think you convinced me. If lazyness is given consideration then its easiest for the consumer to import from one location.